Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -358,18 +362,30 @@ private UniTask ServerSceneLoading(string scenePath, IEnumerable<INetworkPlayer>
 
             SetAllClientsNotReady(players);
 
-            OnServerStartedSceneChange?.Invoke(scenePath, sceneOperation);
 
-            if (Server.LocalClientActive)
+            bool serverSceneIsLoaded = GetSceneByPathOrName(scenePath).isLoaded;
+
+            //Only do a guarantee load if we are loading multiple additive scenes, otherwise only load if unloaded
+            bool serverShouldLoad = loadMultipleAdditiveScenes ? true : !serverSceneIsLoaded;
+
+            if (serverShouldLoad)
+                OnServerStartedSceneChange?.Invoke(scenePath, sceneOperation);
+
+            if (Server.LocalClientActive && serverShouldLoad)
             {
                 // notify host client that scene loading is about to start
                 OnClientStartedSceneChange?.Invoke(scenePath, sceneOperation);
             }
 
             // Coburn, 2023-01-29: Patched the following to not use Forget UniTask functionality.
+            // Moddingdudes, 2023-02-06: This is only for multiple additive, we want just one additive option
             // ALWAYS load the scene, even if we're the host.
             // return the task after sending the message
-            var loadTask = LoadSceneAsync(scenePath, players, sceneOperation, loadSceneParameters);
+
+            UniTask loadTask = UniTask.CompletedTask;
+
+            if (serverShouldLoad)
+                loadTask = LoadSceneAsync(scenePath, players, sceneOperation, loadSceneParameters);
```

Comment:
Looks like loadMultipleAdditiveScenes is doing something different here than in OnServerAuthenticated. 

If it is to load scene on client but not server then it should probably be a new function

Reply:
You're right, adding another variable which separates the logic